qiita_db/metadata_template/util.py

```diff
@@ -25,6 +25,20 @@
     from string import letters, digits
 
 
+def type_lookup(dtype):
+    """Lookup function """
```

Comment:
hahaha sure! I think I haven't finished the docstring >.< too involved on the other issue...